### PR TITLE
Update GYP installer to be promoted in Conan center

### PR DESCRIPTION
**Pull request by uilianries**

### CODE REVIEW

**Review by SSE4 — APPROVED** [review]

File: conanfile.py [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
-    license = "https://chromium.googlesource.com/external/gyp/+/master/LICENSE"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "BSD-3-Clause"
+    settings = "os_build"
     no_copy_source = True
+    _source_subfolder = "source_subfolder"
+
+    def requirements(self):
+        if self.settings.os_build == 'Linux':
+            self.requires.add('glibc_version_header/0.1@bincrafters/stable')
```
Comment (SSE4) [inline conanfile.py]: this is not required, as gyp contains only python code, and we compile nothing

Comment (uilianries) [reply conanfile.py]: okay, will remove it. thanks

Comment (uilianries) [reply conanfile.py]: done!

File: .travis.yml [hunk 2]
```diff
@@ -11,12 +12,15 @@ osx: &osx
 matrix:
    include:
       - <<: *linux
+        env: CONAN_CLANG_VERSIONS=4.0 CONAN_DOCKER_IMAGE=lasote/conanclang40
```
Comment (Croydon) [inline .travis.yml]: It should use the newer `conanio/clang40` container

**Review by Croydon — CHANGES_REQUESTED** [review]
Small change required

File: .travis.yml (hunk 2 above)
Comment (uilianries) [reply .travis.yml]: that's true! thanks!

**Review by Croydon — APPROVED** [review]
LGTM

**Pull request merged**